src/main/java/com/mecatran/gtfsvtor/validation/triptimes/TooFastTravelValidator.java

```diff
@@ -134,9 +134,52 @@ private double getMaxSpeedMps(GtfsRoute route, ValidatorConfig config) {
 		return maxSpeedMps;
 	}
 
+	private int mapExtendedToBaseRouteType(int routeType) {
+		if (routeType >= 100 && routeType < 200) {
+			return GtfsRouteType.RAIL_CODE;
+		} else if (routeType >= 200 && routeType < 300) {
+			return GtfsRouteType.INTERCITY_BUS_CODE;
+		} else if (routeType >= 300 && routeType < 400) {
+			return GtfsRouteType.RAIL_CODE;
+		} else if (routeType >= 400 && routeType < 700 || routeType == 10) {
+			// Google's 10 (Communter train) is not yet an official routeType
+			return GtfsRouteType.METRO_CODE;
+		} else if (routeType >= 700 && routeType < 800) {
+			// Note: Google's Feedvalidator handles 701 (RegionalBus), 702 (ExpressBus)
+			// 715 (Demand and Response Bus Service) and 717 (share taxi service) as
+			// INTERCITY_BUS (=> maxSpeed 120km/h)
```

Comment:
In this note, just to make sure: the feed validator you mention is the Google "internal" one, not the historical python gtfs-validator?
(They are all called "gtfs-validator"... this is the reason I picked-up "gtfsvtor" instead, easier to name and remember)

Reply:
I refer to the python transitfeed feedvalidor with [googletransit extensions](https://github.com/google/transitfeed/blob/master/extensions/googletransit/route.py) enabled.

Reply:
In that case, since we aim for backward-compatibility with this validator, maybe we could map as the comment said? I'm wondering why the code does not so (code from OTP?) My two cents.

Reply:
OTP has no intercity bus mode and thus just maps to bus. transitfeed's feedvalidator defines a maxspeed for the cited route types, which I now map to `INTERCITY_BUS_CODE`